### PR TITLE
Ep 12297 update type declarations

### PR DESCRIPTION
**Pull request by JerryDoubleU**
Updated type declarations so they're up to date and don't cause typing errors in the dashboard apps.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -188,6 +188,24 @@ export interface ThemeSectionProperties {
...
+  widgetType?: WidgetType;
+  zones?: number;
+  position?: number;
+  flip?: boolean;
+}
+
+export interface FeedData {
+  socialItems?: Array<any>;
+  socialFeeds?: Array<any>;
+  trigger: any;
```
Comment (sulco) [inline index.d.ts]: Is it possible to specify the `trigger` type?

File: index.d.ts [hunk 2]
```diff
@@ -188,6 +188,24 @@ export interface ThemeSectionProperties {
...
+export interface Layout {
+  orientation: Orientation;
+  widgetType?: WidgetType;
+  zones?: number;
+  position?: number;
+  flip?: boolean;
+}
+
+export interface FeedData {
+  socialItems?: Array<any>;
```
Comment (sulco) [inline index.d.ts]: Throughout this file we're using `sometype[]` notation (vs `Array<sometype>`) - let's keep it consistent

Comment (sulco) [inline index.d.ts]: If the type of values in these arrays can't be specified on this level I'd still steer away from `any`. 
Would it make sense to make `FeedData` a generic? i.e.
```typescript
interface FeedData<T> {
  socialItems?: T[];
 /* ... */
}
```

**Review by sulco — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (JerryDoubleU) [reply index.d.ts]: added generic type

File: index.d.ts (hunk 2 above)
Comment (JerryDoubleU) [reply index.d.ts]: done

**Review by sulco — APPROVED** [review]

**Pull request merged**